SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -15,12 +16,17 @@ internal class LanguagePatcher
         private const char TextDelimiterOpen = '{';
         private const char TextDelimiterClose = '}';
         private const char KeyValueSeparator = ':';
+        private static readonly Regex OverrideRegex = new Regex("(?<key>[\\w_]+)\\s*:\\s*{(?<value>([~!@#$%^&*()\\-_=+\\[\\];:\"',<>\\/?]|{\\d+}|[\\w\\s\n])+)}(\n|\r\n)*", RegexOptions.Multiline);
```

Comment:
In a regex like this, everything should be made lazy. A greedy quantifier could cause the value to also include the next line.